### PR TITLE
#97/1 - initial screen styles fixes

### PR DESCRIPTION
**Pull request by n0macx**

### CODE REVIEW

File: src/app/view/components/questionForMode.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const questionForMode = (mode) => {
     Staff: "Who is this from Hogwart's staff?",
     Houses: 'To which house do they belong\n?',
   };
-  return elementCreator('h1', {}, `MODE: ${questions[mode]}`);
+  return elementCreator('div', {}, `MODE: ${questions[mode]}`);
```
Comment (vieraboschkova) [inline src/app/view/components/questionForMode.js:9]: Is there a h1 tag on the page somwhere else?

File: static/styles/_vars.scss [hunk 2]
```diff
@@ -7,120 +7,4 @@ $lite-orange: #ff9900f2;
 $orange: #df8600f2;
 $wand-color: #ffd492;
 $button-height: 100px;
-
```
Comment (vieraboschkova) [inline static/styles/_vars.scss:10]: Make sure you remove the unused code before merge

File: static/styles/_vars.scss [hunk 3]
```diff
@@ -7,120 +7,4 @@ $lite-orange: #ff9900f2;
...
-//     grid-column: 1/3;
-//     grid-row: 3/4;
-//   }
-// }
-
-// .header__game-img {
-//   outline: 5px yellow dotted;
-// }
-
+$default-radius: 16px;
```
Comment (vieraboschkova) [inline static/styles/_vars.scss:10]: Good idea to use a variable!

File: static/styles/index.scss [hunk 4]
```diff
@@ -112,16 +123,18 @@
...
 
   .footer {
     grid-column: 1/3;
     grid-row: 3/4;
   }
-
-  .rules-icon {
-    font-size: 1.2em;
+  
+  .halloffame-icon {
```
Comment (vieraboschkova) [inline static/styles/index.scss:135]: Perhaps you could change this class name to more BEM-like

File: src/app/view/components/questionForMode.js (hunk 1 above)
Comment (KatarzynaChmielecka) [reply src/app/view/components/questionForMode.js:9]: Tomek will work on it after work, he has some new idea.

File: static/styles/_vars.scss (hunk 2 above)
Comment (KatarzynaChmielecka) [reply static/styles/_vars.scss:10]: Which part did you mean?

File: static/styles/index.scss (hunk 4 above)
Comment (KatarzynaChmielecka) [reply static/styles/index.scss:135]: > Perhaps you could change this class name to more BEM-like

Ok, we will work on it ;)

File: static/styles/_vars.scss (hunk 2 above)
Comment (vieraboschkova) [reply static/styles/_vars.scss:10]: I mean as a general rule, don't push code that shuoldnt be on development. There is a lot of commented code here that is being removed now.

Comment (KatarzynaChmielecka) [reply static/styles/_vars.scss:10]: In some cases, I don't know how it happened ;). In others, something went on dev, and then the idea was modified, so it is the root of problems.

File: src/app/view/components/questionForMode.js (hunk 1 above)
Comment (n0macx) [reply src/app/view/components/questionForMode.js:9]: Thanks for the comment! There is no other h1 tag, so i changed it from div to h4 tag and will push it today with other fixed stuff.

**Pull request closed without merging**